doc/user/content/sql/functions/cast.md

```diff
@@ -31,29 +31,92 @@ The following special syntax is permitted if _val_ is a string literal:
 
 ### Valid casts
 
-Source type | Return type
-------------|------------
-`bytea`  | `text`
-`int` | `boolean`
-`int` | `float`
-`int` | `decimal`
-`int` | `text`
-`float`| `int`
-`float`| `decimal`<sup>1</sup>
-`float`| `text`
-`decimal` | `int`
-`decimal` | `float`
-`decimal` | `text`
-`date` | `timestamp`
-`date` | `timestamp with time zone`
-`date` | `text`
-`list` | `text`
-`time` | `interval`
-`time` | `text`
-`interval` | `time`
-`interval` | `text`
-`text`  |  `bytea`
-`text` | `list`
+Casts may be:
+
+* **Implicit** -- Values are automatically converted (for example, when you add `int4` to `int8`, the `int4` value is automatically converted to `int8`)
```

Comment:
nit: semantically these are unordered lists, but then you're enforcing an ordering on them.